spec/external_auth_options_spec.rb

```diff
@@ -0,0 +1,67 @@
+describe ManageIQ::ApplianceConsole::ExternalAuthOptions do
+  subject { described_class.new }
+  let(:result) { double(@spec_name, :failure? => false) }
+  let(:rake_set) { "evm:settings:set" }
+
+  before do
+    @spec_name = File.basename(__FILE__).split(".rb").first.freeze
```

Comment:
You don't really need this. You can just give the double a string name like "RakeResult" and it will work fine.

Reply:
@carbonin Thank you for looking this over. Will do.

JoeV